### PR TITLE
fix add breakpoint for compound statement

### PR DESCRIPTION
**Pull request by pahans**

### CODE REVIEW

File: modules/web/js/ballerina/components/compound-statement-decorator.jsx [hunk 1]
```diff
@@ -77,14 +78,43 @@ class CompoundStatementDecorator extends React.Component {
...
                 onJumptoCodeLine={ () => this.onJumptoCodeLine() }
                 onBreakpointClick={ () => this.onBreakpointClick() }
 			/>
+      {		model.isBreakpoint &&
+          this.renderBreakpointIndicator()
+      }
 		</g>);
     }
 
+    onBreakpointClick() {
```
Comment (hemikak) [inline modules/web/js/ballerina/components/compound-statement-decorator.jsx]: Better to have doc comments here.

File: modules/web/js/ballerina/components/compound-statement-decorator.jsx [hunk 2]
```diff
@@ -77,14 +78,43 @@ class CompoundStatementDecorator extends React.Component {
...
+        const { model } = this.props;
+        const { isBreakpoint = false } = model;
+        if(model.isBreakpoint) {
+            model.removeBreakpoint();
+        } else {
+            model.addBreakpoint();
+        }
+    }
+
+    renderBreakpointIndicator() {
```
Comment (hemikak) [inline modules/web/js/ballerina/components/compound-statement-decorator.jsx]: Better to have doc comments here.

File: modules/web/js/ballerina/components/compound-statement-decorator.jsx (hunk 1 above)
Comment (pahans) [inline modules/web/js/ballerina/components/compound-statement-decorator.jsx]: fixed with https://github.com/ballerinalang/composer/pull/2012

File: modules/web/js/ballerina/components/compound-statement-decorator.jsx (hunk 2 above)
Comment (pahans) [inline modules/web/js/ballerina/components/compound-statement-decorator.jsx]: fixed with https://github.com/ballerinalang/composer/pull/2012

**Pull request merged**